goodfood/goodfood/settings.py

```diff
@@ -20,7 +20,7 @@
 # See https://docs.djangoproject.com/en/2.0/howto/deployment/checklist/
 
 # SECURITY WARNING: keep the secret key used in production secret!
-SECRET_KEY = 'arvtrm45njdf%1d+r(4o_&7u8aqg6n!9i#092-lqnwwrfqip_s'
+SECRET_KEY = 'v&k#+#4p=*&7e(u0x-^9icajgtwdhu0@7^8dgmd8a%%*c8#&k7'
```

Comment:
don't push the secret_key; it should be secret ;)
we should keep the old secret_key in an env.